src/app/models/generation_template.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, Integer, String, DateTime
+from sqlalchemy.sql import func
+from app.db.base_class import Base
+
+
+class GenerationTemplate(Base):
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String, index=True)
+    description = Column(String)
+    created_at = Column(DateTime, server_default=func.now())
+    modified_at = Column(DateTime, server_default=func.now(), onupdate=func.now())
+    content = Column(String)
```

Comment:
Please move `content` above `created_at` and `modified_at`
```suggestion
    content = Column(String)
    created_at = Column(DateTime, server_default=func.now())
    modified_at = Column(DateTime, server_default=func.now(), onupdate=func.now())
```

Reply:
Addressed